lib/IDE/CodeCompletion.cpp

```diff
@@ -1448,6 +1451,52 @@ static bool hasTrivialTrailingClosure(const FuncDecl *FD,
   return false;
 }
 
+/// Populates the given DenseMap with a list of overlay modules that should be
+/// treated as if their contents were coming from the module that it shadows.
+/// At present only overlays with a name starting with '_' are added to the map.
+static void populateOverlayToShadowedModuleMapping(SourceFile &SF,
+    llvm::DenseMap<const ModuleDecl*, ModuleDecl*> &OverlayToShadowed) {
```

Comment:
This is also useful from interface generation, isn't it? If so, I think `SourceFile` or `ASTContext` should provide this.